.github/dependabot.yml

```diff
@@ -3,14 +3,18 @@ updates:
   - package-ecosystem: "npm"
     directory: "/front"
     schedule:
-      interval: "weekly"
+      interval: "monthly"
       time: "04:00"
     rebase-strategy: "disabled"
     open-pull-requests-limit: 10
     assignees:
       - "luucvanderzee"
+      - "IvaKop"
```

Comment:
I wonder if we shouldn't extent this to the entire front-end team. But let's give it a shot as is and see if it's even needed.